testsuite/gnat2goto/tests/op_or_else/op_or_else.adb

```diff
@@ -0,0 +1,48 @@
+procedure Op_Or_Else is
+  A : Boolean := True;
+  B : boolean := False;
+  C : boolean := True;
+  
+  --  The following two functions serve the purpose of having
+  --  a division by zero check that serves to demonstrate that
+  --  because of the path followed short circuiting at the call
+  --  site has occured.
```

Comment:
I think this sentence is a bit too complicated. I think 

> These functions are used to test the difference in short circuiting behaviour of `or` and `or else` (`or` doesn't short circuit, `or else` does)

 would express more or less the same thing.